source/iopipe/json/serialize.d

```diff
@@ -695,26 +695,39 @@ private template AllIgnoredMembers(T)
         enum AllIgnoredMembers = staticMap!(ApplyRight!(getUDAs, IgnoredMembers), T, BaseClassesTuple!T);
 }
 
+private enum isJTRef(alias InstantiatedF) = cast(bool) (ParameterStorageClassTuple!InstantiatedF[0] & ParameterStorageClass.ref_);
+
+/// Check if JT is passed by ref for the case where fromJSON already successfully compiled with a policy via IFTI
+private template checkRef(alias FromJSON, JT, P)
```

Comment:
Ugh, I hate that it has to be done this way, but I think you are right, there isn't a better way given how IFTI works.

It's also not really complete, as you could have another unrelated parameter first with a default.

we really need that `__traits(resolve)` mechanism...